app/src/main/java/de/danoeh/antennapod/fragment/FeedItemlistFragment.java

```diff
@@ -106,13 +114,22 @@
     private View header;
     private Toolbar toolbar;
     private ToolbarIconTintManager iconTintManager;
+
+    private SpeedDialView mSpeedDialView;
+    private  LongList checkedIds = new LongList();
```

Comment:
Oh, another copy of the selected items? Please pass them to `EpisodeMultSelectActionHandler` as an argument instead.

Reply:
Ok, yeah, you're correct. That is a good idea. I've extracted the checkedIds to EpisodeMultSelectActionHandler.